plaidml/bridge/openvino/ops/prior_box_clustered.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset5.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;  // NOLINT[build/namespaces]
+using ngraph::opset5::PriorBoxClustered;
+
+namespace PlaidMLPlugin {
+
+void registerPriorBoxClustered() {
+  registerOp("PriorBoxClustered", [](const Context& ctx) {
+    auto* layer = ngraph::as_type<ngraph::opset5::PriorBoxClustered>(ctx.layer);
+    IE_ASSERT(layer);
+    IE_ASSERT(ctx.operands.size() == 2);
+    // According to the size of output, the size of variance shall be restricted
+    const ngraph::op::PriorBoxClusteredAttrs& attrs = layer->get_attrs();
+    IE_ASSERT(attrs.variances.size() == 1 || attrs.variances.size() == 4 || attrs.variances.empty());
+
+    auto precision = to_plaidml(ctx.layer->get_output_element_type(0));
+    int64_t num_priors = static_cast<int64_t>(attrs.widths.size());
+    auto variances = attrs.variances;
+    if (variances.empty()) variances.push_back(0.1f);
+
+    auto input0_shape = cast_constant_operand<int64_t>(0, layer);
+    auto input1_shape = cast_constant_operand<int64_t>(1, layer);
+    int64_t layer_height = static_cast<int64_t>(input0_shape[0]);
```

Comment:
Are these static casts to int64_t necessary?

Reply:
only inputx_shape need to cast, remove others